neurom/fst/tests/test_sectionfunc.py

```diff
@@ -70,13 +71,21 @@ def test_section_area():
 
 
 def test_section_tortuosity():
-    sec_a = Section([
```

Comment:
NeuroM v2 won't have a Section ctor so we need to get rid of its use in the tests.